src/quicr_name.cpp

```diff
@@ -380,18 +356,15 @@ operator<<(std::ostream& os, const Name& name)
 void
 operator<<(messages::MessageBuffer& msg, const Name& val)
 {
-  msg << val.data();
+  msg << val._low;
+  msg << val._hi;
 }
 
 bool
 operator>>(messages::MessageBuffer& msg, Name& val)
 {
-  std::vector<uint8_t> bytes{};
-  if (!(msg >> bytes))
-    return false;
-
-  val = Name{bytes};
-
+  msg >> val._hi;
```

Comment:
Should we put this in network-byte order?

Reply:
The operator>> for uint64_t should be doing that. Unless you mean encode hi then low on the wire?

Reply:
Ah, it is not. Then I guess this questions remains open